### PR TITLE
[feature] #1088 balances page improvements

### PR DESCRIPTION
**Pull request by alanrsoares**
* Add ability to toggle visibility for Zero Balance tokens 
* Add Import token header action (defaults to Rowan on CosmosHub)
* Basic pagination

closes #1088

Default state:
image

Empty state:
image

### CODE REVIEW

**Comment by ajoslin** [comment]
I prefer infinite/virtual scrolling to pagination here.

Also, since cmd-f / ctrl-f will no longer work for finding balances on the screen, we should probably override cmd/ctrl-f to highlight the search input.

File: ui/app/src/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          if (props.pageIndex < props.pages - 1) {
+            props.onPageIndexChange(props.pageIndex + 1);
+          }
+        }}
+      >
+        »
+      </button>
+    </div>
+  );
+}
```
Comment (ajoslin) [inline ui/app/src/components/Pagination/Pagination.tsx:48]: Don't need this component now

Comment (alanrsoares) [reply ui/app/src/components/Pagination/Pagination.tsx]: done

**Comment by ajoslin** [comment]
image
Clicking these should also focus the input (set them as `<label for="inputId">`

**Comment by ajoslin** [comment]
image
When you scroll away from an expanded component it should collapse for you

File: ui/app/src/components/RecyclerView/RecyclerView.tsx [hunk 2]
```diff
@@ -45,55 +40,87 @@ export default defineComponent({
...
-      style: {
-        display: "block",
-        overflowY: "scroll",
-        maxHeight: `${props.rowHeight * props.visibleRows}px`,
-      },
+    onMounted(() => {
+      if (!visibleRows.value && !props.visibleRows) {
+        // dynamically set visibleRows based on the screen height when a default value isn't present
+        visibleRows.value = Math.floor(
+          (document.body.clientHeight * 0.8) / props.rowHeight,
```
Comment (ajoslin) [inline ui/app/src/components/RecyclerView/RecyclerView.tsx:85]: Need to re-calculate on window resize

**Pull request merged**